src/features/Main/components/Footer/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from 'react';
+
+const Footer: FC = () => {
+  return (
+    <div className="bg-[#0F3665] flex flex-wrap justify-between text-white md:px-20 px-8 py-8 text-xs md:text-[15px] md:flex-row flex-col gap-6 md:gap-4 ">
```

Comment:
i think its better to change the position of the copyright to the bottom, so update your code to `md:flex-row flex-col-reverse`